payments-core/src/main/java/com/stripe/android/ConfirmStripeIntentParamsFactory.kt

```diff
@@ -70,30 +73,33 @@ internal class ConfirmPaymentIntentParamsFactory(
     override fun create(
         paymentMethodId: String,
         paymentMethodType: PaymentMethod.Type?,
+        optionsParams: PaymentMethodOptionsParams?,
         requiresSaveOnConfirmation: Boolean,
         recollectedCvc: String?
     ): ConfirmPaymentIntentParams {
+        val fallbackOptions = when (paymentMethodType) {
+            PaymentMethod.Type.Card -> {
+                PaymentMethodOptionsParams.Card(
+                    cvc = recollectedCvc,
+                    setupFutureUsage = ConfirmPaymentIntentParams.SetupFutureUsage.OffSession?.takeIf {
+                        requiresSaveOnConfirmation
```

Comment:
This is actually quite hard to pipe in the "right" way. But it's only used for link inline in passthrough mode, as far as I can tell.